example/build.gradle

```diff
@@ -6,8 +6,13 @@ repositories {
 
 dependencies {
     compile project(':stripe')
-    compile 'com.android.support:appcompat-v7:22.2.1'
-    compile 'com.android.support:support-v4:24.1.1'
+    compile 'com.android.support:appcompat-v7:25.0.1'
```

Comment:
This is all in the example project. Not forcing upgrades or extra inclusion of new third party libraries for anyone that doesn't want to use them.